app/index.js

```diff
@@ -1,72 +1,10 @@
 require('dotenv').config();
-var bole = require('bole');
-var config = require('./config.js');
-var express = require('express');
-var join = require('path').join;
-var nunjucks = require('nunjucks');
-var passport = require('passport');
-var raven = require('raven');
-var Strategy = require('passport-auth0-openidconnect').Strategy;
-var assets = require('./assets');
+const bole = require('bole');
 
+const config = require('./config');
+const create_app = require('./factory');
```

Comment:
As discussed offline you may want to rename this module to `create_app` OR export using `module.exports.create_app = () => {}` and then here change this to `const create_app = require('./app_factory').create_app;`.

BTW: I like how you moved things around, it makes things a bit less daunting and easier to follow 👍